internal/config/config.go

```diff
@@ -154,7 +155,7 @@ func initConfig() {
 		viper.SetDefault("RBAC_ENABLE", false)
 
 		// prometheus config
-		viper.SetDefault("PROMETHEUS_PORT", "5005")
+		viper.GetString("PROMETHEUS_PORT")
```

Comment:
Can we keep some default value for this as it was before? Some OS allow binding multiple service on same port.